inc/Classes/Translation.php

```diff
@@ -4,25 +4,32 @@
 
 /**
  * Global translation
+ *
+ * TODO Implement proper caching interface
+ * TODO Implement dependency injection to remove `global` keywords
  */
 class Translation
 {
     /**
-     * Global language
-     *
-     * @var string
+     * Global language.
+     */
+    private string $language = 'de';
+
+    /**
+     * Default language.
+     * Will be used in cases where an invalid language was selected.
      */
-    public $language = 'de';
+    private string $defaultLanguage = 'de';
```

Comment:
Hmm. If this is a value that is never changed...should we make this a constant?

Reply:
I would suggest the second option (making this configurable).
I created a ticket for this: https://github.com/lansuite/lansuite/issues/670